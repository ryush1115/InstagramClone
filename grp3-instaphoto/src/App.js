import React from 'react'
import '../node_modules/bootstrap/dist/css/bootstrap.min.css'
import './App.css'
import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom'
import Login from './components/login.component'
import SignUp from './components/signup.component'
import Userprofile from './components/userprofile.component'

function App() {
  return (
    <Router>
      <div className="App">
        <nav className="navbar navbar-expand-lg navbar-light fixed-top">
          <div className="container">
            <Link className="navbar-brand" to={'/sign-in'}>
              Instaphoto
            </Link>
            <div className="collapse navbar-collapse" id="navbarTogglerDemo02">
              <ul className="navbar-nav ml-auto">
                <li className="nav-item">
                  <Link className="nav-link" to={'/sign-in'}>
                    Login
                  </Link>
                </li>
                <li className="nav-item">
                  <Link className="nav-link" to={'/sign-up'}>
                    Sign up
                  </Link>
                </li>
                <li className="nav-item">
                  <Link className="nav-link" to={'/user-profile'}>
                    User Profile
                  </Link>
                </li>
              </ul>
            </div>
          </div>
        </nav>
        {/* <div className="auth-wrapper"> */}
        <div>  
          <div>
          {/* <div className="auth-inner"> */}
            <Routes>
              <Route exact path="/" element={<Login />} />
              <Route path="/sign-in" element={<Login />} />
              <Route path="/sign-up" element={<SignUp />} />
              <Route path="/user-profile" element={<Userprofile />} />              
            </Routes>
          </div>
        </div>
      </div>
    </Router>
  )
}
export default App