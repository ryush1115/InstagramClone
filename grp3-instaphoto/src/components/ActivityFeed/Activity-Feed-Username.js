import React from 'react';
import '../../activityfeed.css';
import '../../userprofile.css';

export default function ActivityFeedUsername(props) {

    const username = props.username;

    return (
        <div className="profile-user-settings">
            <h1 className="profile-user-name">{username}</h1>
        </div>
    );
}
