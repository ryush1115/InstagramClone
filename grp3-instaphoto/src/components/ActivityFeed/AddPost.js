// import React from 'react'
import FilterablePostTable from "./FilterablePostTable";
import React, { useState,useRef } from "react";

export default function AddPost() {
    // local state new Post
    // we don't need the state variable since we are not passing it as a prop
    // Its child will get the list of Posts from the backend

    const [, setNewPost] = useState(null);

    // Ref variable to tell the FilterablePostTable
    // to load data or not
    const loadData = useRef(false);

    // let newUsername;
    // let newPostComment;
    // let newPostImage = "http://loremflickr.com/640/480"; // default to this image for HW2

    return (
      <div>
        {' '}
        <FilterablePostTable reload={loadData} />
        <div>
        </div>
      </div>
    )

}
