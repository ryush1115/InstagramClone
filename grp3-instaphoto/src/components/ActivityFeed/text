This is a text file because git disallows empty directories.