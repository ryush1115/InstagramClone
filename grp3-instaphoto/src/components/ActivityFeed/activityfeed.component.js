/* eslint-disable no-template-curly-in-string */
import React, { useState, Fragment, useEffect, useRef } from "react";
import './activityfeed.css';
import '../UserProfile/userprofile.css';

import AddPost from "./AddPost";

import ActivityFeedUsername from "./Activity-Feed-Username";
import Sidebar from './../sidebar.component'

// import {  createComment} from '../../api/mock_api';
// import FilterablePostTable from "./FilterablePostTable";

const ActivityFeedComponent = () => {
  const[, setNewComment] = useState(null);
  const[, setDeletedPost] = useState(null);
  const[, setIncrementLike] = useState(null);
  const [roster, setRoster] = useState([]);
  const [create, setCreate] = useState('default');


  // function PostTable(props) {
  //   // counter to provide unique key to rows
  //   const counter = useRef(0);
  //   let postsList = [];
  //   let usernameFilter = 'SHOW_ALL';
  //   // get the list of posts and the username from props

  //   if (props.username) {
  //     usernameFilter = props.username;
  //   }
  //   postsList = props.posts;

  //   const makeRows = () => {
  //     const rows = [];

  //     postsList.forEach((element) => {
  //       // const {post} = element;
  //       if (usernameFilter === 'SHOW_ALL') {
  //         rows.unshift(
  //           <PostRow post={element}
  //             key={counter.current}
  //           />
  //         );
  //       } else {
  //         if (!element.username.startsWith(usernameFilter)) {
  //           return;
  //         }
  //         rows.unshift(
  //           <PostRow post={element}
  //             key={counter.current}
  //           />,
  //         );
  //       }
  //       // increment counter
  //       counter.current += 1;

  //     });
  //     return rows;
  //   };

  //   const rows = makeRows();
  //   return (
  //     <table>
  //       <tbody>{rows}</tbody>

  //     </table>

  //   );
  // }

  /**
   * Searchbar component
   * pass the username entered as props to its children
   * @param {} props
   */

  // function SearchBar(props) {
  //   const [username, setUsername] = useState('');
  //   const handleFilterTextChange = (e) => {
  //     setUsername(e.target.value);
  //   };

  //   return (
  //     <div>
  //       {/* <form>
  //         <input
  //           type="text"
  //           placeholder="filter by username..."
  //         />
  //       </form> */}
  //       <PostTable username={username} posts={props.roster} />
  //     </div>
  //   )
  // }

  /**
   * FilterablePostsTable component
   * This component fetched the list of Posts fromthe
   * backend and pass it as props to its child
   * @param {*} props
   * @returns FilterablePostsTable elements
   */

  // function FilterablePostTable(props) {
  //   // Local state to store and update the list of Posts
  //   const [roster, setRoster] = useState([]);
  //   // ref to indicate if this is the first rendering
  //   const firstRendering = useRef(true);
  //   // get the list of [Timeline] Posts from the backend
  //   useEffect(() => {
  //     // get the list of [Timeline] Posts from the backend
  //     async function fetchData() {
  //       const data = await getPosts();
  //       setRoster(data);
  //     }

  //     // only load data on the first rendering or
  //     // when a new post is created
  //     if (firstRendering.current || props.reload.current) {
  //       firstRendering.current = false;
  //       props.reload.current = false; // set reload to false
  //       fetchData();
  //     }

  //   });

  //   return (
  //     <SearchBar roster={roster} />
  //   )
  // }

  // function AddComment() {
  //   // local state new Comment
    
  //   const[, setNewComment] = useState(null);

  //   // Ref variable 
  //   const loadData = useRef(false);

  //   let newPostComment_;
  //   let tagOfOtherUsers;

  //   const handleOnChangeComment = (e) => {
  //     if (e.target.name==='commentBox_')  {
  //       newPostComment_ = e.target.value;
  //     }
  //   }

  //   const handleCreateComment = async (e) => {
  //     // stop default behavior to avoid reloading the page
  //     e.preventDefault();
  //     const newComment = {username:"grp3foreva", message:"my first comment", tagOfOtherUsers:null};
      
  //     // clear the form
  //     const form = document.getElementByIdById('commentBox');
  //     form.reset();
  //     const newStoredComment = await createComment(newComment); 
  //     // update LoadData
  //     loadData.current = true;
  //     setNewComment(newStoredComment);
  //   }
  // }

  // function AddPost() {
  //   // local state new Post
  //   // we don't need the state variable since we are not passing it as a prop
  //   // Its child will get the list of Posts from the backend

  //   const [, setNewPost] = useState(null);

  //   // Ref variable to tell the FilterablePostTable
  //   // to load data or not
  //   const loadData = useRef(false);

  //   let newUsername;
  //   let newPostComment;
  //   let newPostImage = "http://loremflickr.com/640/480"; // default to this image for HW2

  //   return (
  //     <div>
  //       {' '}
  //       <FilterablePostTable reload={loadData} />
  //       <div>
  //       </div>
  //     </div>
  //   )
  // }

  return (
    <Fragment>
      <Sidebar create = {create} setCreate = {setCreate}/>
      <header>
        <div className="container" data-testid = "testcc">
          <div className="profile">
            <div className="profile-image">
              <img alt="" />
              {/* <img src={require('../images/grp3.PNG')} alt=""/> */}

            </div>
            <ActivityFeedUsername username={"grp3foreva"}/>
          </div>
        </div>
      </header>

      <main>

        <div className="container">
          <div className="feed">
            <div className="feedWrapper">
              <AddPost />
            </div>
          </div>
        </div>
      </main>
    </Fragment>
  )
}
export default ActivityFeedComponent;
