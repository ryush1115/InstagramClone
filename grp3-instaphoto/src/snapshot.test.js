/**
* @jest-environment jsdom
*/

// import testing library functions
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
// import testing library functions
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import renderer from 'react-test-renderer';
import {within} from '@testing-library/dom'
import WeatherApp from './App';


import App from './App';
import Login from './components/login.component'
import SignUp from './components/signup.component'
import Userprofile from './components/userprofile.component'
import ActivityFeed from './components/activityfeed.component'
import FriendSuggestion from './components/FriendSuggestion.component'

test('Weather app matches snapshot', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });