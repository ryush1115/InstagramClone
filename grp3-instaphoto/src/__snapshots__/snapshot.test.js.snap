// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activity Feed matches snapshot 1`] = `
Array [
  <header>
    <div
      class="container"
    >
      <div
        class="profile"
      >
        <div
          class="profile-image"
        >
          <img
            alt=""
          />
        </div>
        <div
          class="profile-user-settings"
        >
          <h1
            class="profile-user-name"
          >
            grp3foreva
          </h1>
        </div>
      </div>
    </div>
  </header>,
  <main>
    <div
      class="container"
    >
      <div
        class="feed"
      >
        <div
          class="feedWrapper"
        >
          <div>
             
            <div>
              <table>
                <tbody />
              </table>
            </div>
            <div />
          </div>
        </div>
      </div>
    </div>
  </main>,
]
`;

exports[`App matches snapshot 1`] = `
<div
  className="App"
>
  <nav
    className="navbar navbar-expand-lg navbar-light fixed-top"
  >
    <div
      className="container"
    >
      <a
        className="navbar-brand"
        href="/sign-in"
        onClick={[Function]}
      >
        Instaphoto
      </a>
      <div
        className="collapse navbar-collapse"
        id="navbarTogglerDemo02"
      >
        <ul
          className="navbar-nav ml-auto"
        >
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/sign-in"
              onClick={[Function]}
            >
              Login
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/sign-up"
              onClick={[Function]}
            >
              Sign up
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/userprofile"
              onClick={[Function]}
            >
              User Profile
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/activity-feed"
              onClick={[Function]}
            >
              Activity Feed
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/FriendSuggestion"
              onClick={[Function]}
            >
              Friend Suggestion
            </a>
          </li>
          <li
            className="createpost"
          >
            <a
              className="nav-link"
              href="/dragdrop"
              onClick={[Function]}
            >
              Create Post
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/FollowingList"
              onClick={[Function]}
            >
              Following List
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
  <div>
    <div>
      <div
        className="auth-wrapper"
      >
        <form
          className="auth-inner"
        >
          <div
            className="header"
          >
            <h3>
              Welcome to Instaphoto 
            </h3>
          </div>
          <br />
          <div
            className="login-and-password"
          >
            <label>
              Email
            </label>
            <input
              className="form-control"
              onChange={[Function]}
              placeholder="Enter email"
              type="email"
              value=""
            />
          </div>
          <div
            className="login-and-password"
          >
            <label>
              Password
            </label>
            <p>
              Forgot password?
            </p>
            <input
              className="form-control"
              onChange={[Function]}
              placeholder="Enter password"
              type="password"
              value=""
            />
          </div>
          <br />
          <div
            className="d-grid"
          >
            <button
              className="btn btn-primary"
              onClick={[Function]}
              type="submit"
            >
              Sign in
            </button>
          </div>
          <p
            className="login-create-account text-right"
          >
            Don't have an account? 
            <br />
            <a
              href="#"
              onClick={[Function]}
            >
               Sign up now to join communities across the globe 
            </a>
          </p>
        </form>
      </div>
    </div>
  </div>
</div>
`;

exports[`CreatePost matches snapshot 1`] = `
Array [
  <div
    className="create-post"
  >
    <div
      className="create-post_content"
    >
      <div
        className="create-post_container"
      >
        <div
          className="create-post_title"
        >
          Create new post
        </div>
        <div
          className="create-post_close"
        >
          <img
            alt="close"
            src="https://static.xx.fbcdn.net/rsrc.php/v3/y2/r/__geKiQnSG-.png"
          />
        </div>
        <div>
          <div
            className="dropzone"
            onBlur={[Function]}
            onClick={[Function]}
            onDragEnter={[Function]}
            onDragLeave={[Function]}
            onDragOver={[Function]}
            onDrop={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            role="presentation"
            tabIndex={0}
          >
            <input
              className="input-zone"
              multiple={true}
              onChange={[Function]}
              onClick={[Function]}
              style={
                Object {
                  "display": "none",
                }
              }
              tabIndex={-1}
              type="file"
            />
            <div
              className="text-center"
            >
              <p
                className="dropzone-content"
              >
                Drag’ n’ drop some files here, or click to select files
              </p>
              <button
                className="btn"
                type="button"
              >
                Click to select files
              </button>
            </div>
          </div>
          <form
            id="commentBox"
          >
            <input
              className="commentBox"
              placeholder="Enter a comment..."
              size="20"
              type="text"
            />
          </form>
          <aside>
            <ul />
          </aside>
        </div>
      </div>
    </div>
  </div>,
  ",",
]
`;

exports[`Dragdrop matches snapshot 1`] = `
<div
  className="create-post_"
>
  <div
    className="create-post-content_"
  >
    <div
      className="create-post_title_"
    >
      Create a new post!
    </div>
    <form
      id="form-file-upload"
      onSubmit={[Function]}
    >
      <p>
        Create a Post Here!! 
      </p>
      <input
        name="comment"
        onChange={[Function]}
        placeholder="Enter a comment..."
        type="text"
      />
      <input
        id="postImage"
        name="postImage"
        onChange={[Function]}
        type="file"
      />
      <button
        type="submit"
      >
        Create Post
      </button>
    </form>
  </div>
</div>
`;

exports[`FollowingList matches snapshot 1`] = `
Array [
  <div
    className="container-0_FriendSuggestion"
  >
    <span
      className="text-0_FriendSuggestion"
      data-testid="followbutton"
    >
      Following
    </span>
  </div>,
  ",",
]
`;

exports[`FriendSuggestion matches snapshot 1`] = `
Array [
  <nav
    className="home navbar navbar-expand navbar-light"
  >
    <div
      className="card card"
    >
      <div
        className="logo"
      >
        <h>
          Instaphoto 
        </h>
        <img
          alt="logo"
        />
      </div>
      <br />
      <div>
        <img
          className="card-img-top sig"
        />
        <p
          className="username"
        >
           grp3foreva
        </p>
      </div>
      <div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            <div
              className="space"
            />
            <div
              className="list-group list-group-flush"
              onKeyDown={[Function]}
            >
              <div
                className="list list-group-item"
                data-rr-ui-event-key={null}
                onClick={[Function]}
              >
                <span>
                  Home
                </span>
              </div>
              <div
                className="list list-group-item"
                data-rr-ui-event-key={null}
                onClick={[Function]}
              >
                <span>
                  Create
                </span>
              </div>
              <div
                className="list list-group-item"
                data-rr-ui-event-key={null}
                onClick={[Function]}
              >
                <span>
                  Profile
                </span>
              </div>
            </div>
          </p>
        </div>
      </div>
      <br />
      <div
        className="suggestions"
      >
        <p
          className="suggestion-text"
        >
          Suggestions for you
        </p>
        <button
          className="show-all-btn"
        >
          See all
        </button>
        <div
          className="profile-card"
        >
          <div>
            <img
              alt=""
              className="other-user"
            />
          </div>
          <button
            className="action-btn"
            type="button"
          >
            <span
              className="username"
            >
              akikozzm
            </span>
            <span
              className="follow"
            >
              Follow
            </span>
          </button>
        </div>
        <div
          className="profile-card"
        >
          <div>
            <img
              alt=""
              className="other-user"
            />
          </div>
          <button
            className="action-btn"
            type="button"
          >
            <span
              className="username"
            >
              akikozzm
            </span>
            <span
              className="follow"
            >
              Follow
            </span>
          </button>
        </div>
        <div
          className="profile-card"
        >
          <div>
            <img
              alt=""
              className="other-user"
            />
          </div>
          <button
            className="action-btn"
            type="button"
          >
            <span
              className="username"
            >
              akikozzm
            </span>
            <span
              className="follow"
            >
              Follow
            </span>
          </button>
        </div>
      </div>
    </div>
  </nav>,
  <div />,
  <div
    className="container-0_FriendSuggestion"
  >
    <span
      className="text-0_FriendSuggestion"
    >
      Suggested
    </span>
  </div>,
  ",",
]
`;

exports[`Login matches snapshot 1`] = `
Array [
  <div
    className="auth-wrapper"
  >
    <form
      className="auth-inner"
    >
      <div
        className="header"
      >
        <h3>
          Welcome to Instaphoto 
        </h3>
      </div>
      <br />
      <div
        className="login-and-password"
      >
        <label>
          Email
        </label>
        <input
          className="form-control"
          onChange={[Function]}
          placeholder="Enter email"
          type="email"
          value=""
        />
      </div>
      <div
        className="login-and-password"
      >
        <label>
          Password
        </label>
        <p>
          Forgot password?
        </p>
        <input
          className="form-control"
          onChange={[Function]}
          placeholder="Enter password"
          type="password"
          value=""
        />
      </div>
      <br />
      <div
        className="d-grid"
      >
        <button
          className="btn btn-primary"
          onClick={[Function]}
          type="submit"
        >
          Sign in
        </button>
      </div>
      <p
        className="login-create-account text-right"
      >
        Don't have an account? 
        <br />
        <a
          href="#"
          onClick={[Function]}
        >
           Sign up now to join communities across the globe 
        </a>
      </p>
    </form>
  </div>,
  ",",
]
`;

exports[`SideBar matches snapshot 1`] = `
<nav
  className="home navbar navbar-expand navbar-light"
>
  <div
    className="card card"
  >
    <div
      className="logo"
    >
      <h>
        Instaphoto 
      </h>
      <img
        alt="logo"
      />
    </div>
    <br />
    <div>
      <img
        className="card-img-top sig"
      />
      <p
        className="username"
      >
         grp3foreva
      </p>
    </div>
    <div>
      <div
        className="card-body"
      >
        <p
          className="card-text"
        >
          <div
            className="space"
          />
          <div
            className="list-group list-group-flush"
            onKeyDown={[Function]}
          >
            <div
              className="list list-group-item"
              data-rr-ui-event-key={null}
              onClick={[Function]}
            >
              <span>
                Home
              </span>
            </div>
            <div
              className="list list-group-item"
              data-rr-ui-event-key={null}
              onClick={[Function]}
            >
              <span
                onClick={[Function]}
              >
                Create
              </span>
            </div>
            <div
              className="list list-group-item"
              data-rr-ui-event-key={null}
              onClick={[Function]}
            >
              <span>
                Profile
              </span>
            </div>
          </div>
        </p>
      </div>
    </div>
    <br />
    <div
      className="suggestions"
    >
      <p
        className="suggestion-text"
      >
        Suggestions for you
      </p>
      <button
        className="show-all-btn"
      >
        See all
      </button>
      <div
        className="profile-card"
      >
        <div>
          <img
            alt=""
            className="other-user"
          />
        </div>
        <button
          className="action-btn"
          type="button"
        >
          <span
            className="username"
          >
            akikozzm
          </span>
          <span
            className="follow"
          >
            Follow
          </span>
        </button>
      </div>
      <div
        className="profile-card"
      >
        <div>
          <img
            alt=""
            className="other-user"
          />
        </div>
        <button
          className="action-btn"
          type="button"
        >
          <span
            className="username"
          >
            akikozzm
          </span>
          <span
            className="follow"
          >
            Follow
          </span>
        </button>
      </div>
      <div
        className="profile-card"
      >
        <div>
          <img
            alt=""
            className="other-user"
          />
        </div>
        <button
          className="action-btn"
          type="button"
        >
          <span
            className="username"
          >
            akikozzm
          </span>
          <span
            className="follow"
          >
            Follow
          </span>
        </button>
      </div>
    </div>
  </div>
</nav>
`;

exports[`SignUp matches snapshot 1`] = `
Array [
  <form
    className="auth-inner"
    onSubmit={[Function]}
  >
    <div
      className="header"
    >
      <h3>
        Welcome to Instaphoto 
      </h3>
    </div>
    <br />
    <div
      className="login-and-password"
    >
      <h3>
         Signup
      </h3>
    </div>
    <div
      className="login-and-password"
    >
      <label>
        Username
      </label>
      <input
        className="form-control"
        name="username"
        onChange={[Function]}
        placeholder="Username"
        type="text"
      />
    </div>
    <div
      className="login-and-password"
    >
      <label>
        Email
      </label>
      <input
        className="form-control"
        name="email"
        onChange={[Function]}
        placeholder="Email"
        type="email"
      />
    </div>
    <div
      className="login-and-password"
    >
      <label>
        Password
      </label>
      <input
        className="form-control"
        name="password"
        onChange={[Function]}
        placeholder="Password (more than 8 characters)"
        type="password"
      />
    </div>
    <div
      className="login-and-password"
    >
      <label>
        Re-enter Password
      </label>
      <input
        className="form-control"
        name="confirmPassword"
        onChange={[Function]}
        placeholder="Re-enter password"
        type="password"
      />
    </div>
    <br />
    <div
      className="d-grid"
    >
      <button
        className="btn btn-primary"
        type="submit"
      >
        Finish
      </button>
    </div>
    <p
      className="forgot-password text-right"
    >
      Already registered 
      <a
        href="/sign-in"
        onClick={[Function]}
      >
        sign in?
      </a>
    </p>
  </form>,
  ",",
]
`;

exports[`UserProfile matches snapshot 1`] = `
Array [
  <nav
    className="home navbar navbar-expand navbar-light"
  >
    <div
      className="card card"
    >
      <div
        className="logo"
      >
        <h>
          Instaphoto 
        </h>
        <img
          alt="logo"
        />
      </div>
      <br />
      <div>
        <img
          className="card-img-top sig"
        />
        <p
          className="username"
        >
           grp3foreva
        </p>
      </div>
      <div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            <div
              className="space"
            />
            <div
              className="list-group list-group-flush"
              onKeyDown={[Function]}
            >
              <div
                className="list list-group-item"
                data-rr-ui-event-key={null}
                onClick={[Function]}
              >
                <span>
                  Home
                </span>
              </div>
              <div
                className="list list-group-item"
                data-rr-ui-event-key={null}
                onClick={[Function]}
              >
                <div
                  className="create-hover"
                >
                  <span
                    onClick={[Function]}
                  >
                    Create
                  </span>
                </div>
              </div>
              <div
                className="list list-group-item"
                data-rr-ui-event-key={null}
                onClick={[Function]}
              >
                <span>
                  Profile
                </span>
              </div>
            </div>
          </p>
        </div>
      </div>
      <br />
      <div
        className="suggestions"
      >
        <p
          className="suggestion-text"
        >
          Suggestions for you
        </p>
        <button
          className="show-all-btn"
          onClick={[Function]}
        >
          See all
        </button>
        <div
          className="profile-card"
        >
          <div>
            <img
              alt=""
              className="other-user"
            />
          </div>
          <button
            className="action-btn"
            type="button"
          >
            <span
              className="username"
            >
              akikozzm
            </span>
            <span
              className="follow"
            >
              Follow
            </span>
          </button>
        </div>
        <div
          className="profile-card"
        >
          <div>
            <img
              alt=""
              className="other-user"
            />
          </div>
          <button
            className="action-btn"
            type="button"
          >
            <span
              className="username"
            >
              akikozzm
            </span>
            <span
              className="follow"
            >
              Follow
            </span>
          </button>
        </div>
        <div
          className="profile-card"
        >
          <div>
            <img
              alt=""
              className="other-user"
            />
          </div>
          <button
            className="action-btn"
            type="button"
          >
            <span
              className="username"
            >
              akikozzm
            </span>
            <span
              className="follow"
            >
              Follow
            </span>
          </button>
        </div>
      </div>
    </div>
  </nav>,
  <header>
    <div
      class="container"
    >
      <div
        class="profile"
      >
        <div
          class="profile-image"
        >
          <img
            alt=""
          />
        </div>
        <div
          class="profile-user-settings"
        >
          <h1
            class="profile-user-name"
          >
            grp3foreva
          </h1>
        </div>
        <div
          class="profile-stats"
        >
          <ul>
            <li>
              <span
                class="profile-stat-count"
              >
                39
              </span>
               posts
            </li>
            <li>
              <span
                class="profile-stat-count"
              >
                169
              </span>
               followers
            </li>
            <li
              onClick={[Function]}
            >
              <span
                class="profile-stat-count"
              >
                667
              </span>
               following
            </li>
          </ul>
        </div>
      </div>
    </div>
  </header>,
  <main>
    <div
      class="container"
    >
      <div
        class="gallery"
      >
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1511765224389-37f0e77cf0eb?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1497445462247-4330a224fdb1?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1426604966848-d7adac402bff?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1502630859934-b3b41d18206c?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1498471731312-b6d2b8280c61?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1515023115689-589c33041d3c?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1504214208698-ea1916a2195a?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1515814472071-4d632dbc5d4a?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1511407397940-d57f68e81203?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1518481612222-68bbe828ecd1?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1505058707965-09a4469a87e4?w=500&h=500&fit=crop"
          />
        </div>
        <div
          class="gallery-item"
        >
          <img
            alt=""
            class="gallery-image"
            src="https://images.unsplash.com/photo-1423012373122-fff0a5d28cc9?w=500&h=500&fit=crop"
          />
        </div>
      </div>
    </div>
  </main>,
  ",",
]
`;
